src/main/java/org/alliancegenome/curation_api/base/entity/AuditedObject.java

```diff
@@ -47,11 +48,13 @@ public class AuditedObject extends BaseEntity {
     @FullTextField(analyzer = "autocompleteAnalyzer", searchAnalyzer = "autocompleteSearchAnalyzer", valueBridge = @ValueBridgeRef(type = OffsetDateTimeValueBridge.class))
     @KeywordField(name = "dateCreated_keyword", sortable = Sortable.YES, searchable = Searchable.YES, aggregable = Aggregable.YES, valueBridge = @ValueBridgeRef(type = OffsetDateTimeValueBridge.class))
     @JsonView({View.FieldsOnly.class})
+    @CreationTimestamp
```

Comment:
I'm not sure we want to use this.  It overwrites any submitted dateCreated value with the time of the load.

Reply:
So in setting the dateCreated manually this annotation overrides it?

Reply:
Because the alternative is to have to have code everywhere that sets this, or just nulls all over the database.

Reply:
Yeah, it overwrites submitted values.  I've not found a way to set it to only change null values.  I agree that having to set it all over is not ideal, but maybe nulls are acceptable for a lot of classes?

Reply:
So I think I found a compromise not sure if it works: `@ColumnDefault("current_timestamp")` its suppose to only set it if the value coming in is null.

Reply:
Yeah, that should do what we want